### ISSUE TITLE
Suggestion: In-tree demo

### ISSUE THREAD

**Issue by pfalcon** [haltu/muuri#107]
There're few demo-related tickets, but I don't immediately see subj request, so dare to open another ticket.

So, the README is pretty good on describing the steps to bootstrap usage of Muuri, and one can use "Show source" on the demo on the site, or even that JSFiddle work-alike, which spends 80% of vertical height on the codes and 20% on the actual demo.

But nothing above can beat the oldskul way of cloning the repo, finding an *.html file to tap Enter on to have it open in the browser, then fire one's favorite editor on it, and learn/hack away from that.

**Comment by niklasramo** [comment]
Yep, totally understand your point and there actually used to be some demos in the repo, but I removed those in favour of having them on CodePen. Maybe I'll add a demo or two to the repo, let's see.

**Comment by pfalcon** [comment]
Thanks for consideration!
